go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/ONSdigital/dp-code-list-api
 go 1.19
 
 require (
-	github.com/ONSdigital/dp-graph/v2 v2.7.2
-	github.com/ONSdigital/dp-healthcheck v1.3.0
-	github.com/ONSdigital/dp-net/v2 v2.6.0
-	github.com/ONSdigital/log.go/v2 v2.2.0
+	github.com/ONSdigital/dp-graph/v2 v2.15.0
+	github.com/ONSdigital/dp-healthcheck v1.5.1-0.20230419121137-48b17cfe6eab
+	github.com/ONSdigital/dp-net/v2 v2.9.0
+	github.com/ONSdigital/log.go/v2 v2.3.0
```

Comment:
Can you please wait until I finish sorting the v1/v2 dependency mess before upgrading these?